### PR TITLE
add GoodDollar

### PR DESCRIPTION
**Pull request by sirpy**
> To make a submission to the Dapps Page, please complete the task lists
> below; otherwise remove them.

Your Dapp:

- [x] connects to Valora via WalletConnect V1 or DAppKit;
- [x] must be mobile-first or mobile responsive;
- [x] is in one of the existing categories; and
- [x] includes a link to user documentation.

### CODE REVIEW

**Comment by silasbw** [comment]
Thank you @sirpy.

The Test might be failing if your IDE format/lint settings are conflicting with the ones in this repo.  you can run `yarn lint && yarn format` to automatically fix them.

We're excited to add GoodDollar!

File: locales/base.json [hunk 1]
```diff
@@ -50,7 +50,8 @@
     "toucan": "Tokenized carbon credits for climate action at scale",
     "cred": "Get your free web3 Credit Score",
     "kolor": "Access for conservation of Patagonian lands from everywhere",
-    "swftpro": "Bridge assets onto BTC/ETH/BSC/CELO/AVAX/+ safely and securely"
+    "swftpro": "Bridge assets onto BTC/ETH/BSC/CELO/AVAX/+ safely and securely",
+    "gooddollar": "Get free crtypo universal basic income. Stake, save and earn while supporting UBI"
```
Comment (silasbw) [inline locales/base.json:54]: We have a few unpublished guidelines on description copy (we're working to share them).  Does this phrasing work for you?

> Claim UBI daily or stake to earn and support UBI

Comment (sirpy) [inline locales/base.json:54]:
```suggestion
    "gooddollar": "Claim UBI daily or stake to earn and support UBI"
```

File: locales/base.json [hunk 2]
```diff
@@ -89,6 +90,7 @@
     "toucan": "Use your crypto to create carbon credits for climate action",
     "cred": "Get your Web3 credit score",
     "kolor": "Access for conservation of Patagonian lands from everywhere",
-    "swftpro": "Bridge assets onto BTC/ETH/BSC/CELO/AVAX/+ safely and securely"
+    "swftpro": "Bridge assets onto BTC/ETH/BSC/CELO/AVAX/+ safely and securely",
+    "gooddollar": "Get free crtypo universal basic income. Stake, save and earn while supporting UBI"
```
Comment (sirpy) [inline locales/base.json:94]:
```suggestion
    "gooddollar": "Claim UBI daily or stake to earn and support UBI"
```

**Comment by MuckT** [comment]
@sirpy it looks like there's an issue with the asset sizing. The `gooddollar.png` assets should be 256px by 256px.

**Comment by sirpy** [comment]
@MuckT done

**Review by silasbw — APPROVED** [review]

**Comment by silasbw** [comment]
Awesome, ty @sirpy :100: !

**Pull request merged**